src/GitHook/Command/FileCommand/PreCommit/PhpStanCheckCommand.php

```diff
@@ -92,23 +92,40 @@ protected function isFileAllowed(string $filePath, array $allowedDirectories): b
     protected function getLevel(PhpStanConfiguration $phpStanConfiguration, CommandContextInterface $context): int
     {
         $level = $phpStanConfiguration->getLevel();
+        $phpStanConfigFilePath = $this->findPhpStanJson($context->getFile());
 
-        if (!$context->isModuleFile()) {
+        if (!$phpStanConfigFilePath) {
             return $level;
         }
 
-        $modulePath = $context->getModulePath();
-        $modulePhpstanConfigurationPath = $modulePath . 'phpstan.json';
-
-        if (!file_exists($modulePhpstanConfigurationPath)) {
-            return $level;
-        }
-
-        $moduleConfig = json_decode(file_get_contents($modulePhpstanConfigurationPath), true);
+        $moduleConfig = json_decode(file_get_contents($phpStanConfigFilePath), true);
         if (!isset($moduleConfig['defaultLevel'])) {
             return $level;
         }
 
         return $moduleConfig['defaultLevel'];
     }
+
+    /**
+     * @param string $filePath
+     *
+     * @return bool|string
+     */
+    protected function findPhpStanJson(string $filePath)
+    {
+        $srcDirectoryPosition = strrpos($filePath, '/src/');
```

Comment:
If we need to support windows, you must use DS constant :) otherwise the element wouldnt be found there.